Source/WebKit/UIProcess/ios/WKMouseGestureRecognizer.mm

```diff
@@ -90,8 +92,12 @@ - (BOOL)_shouldReceiveTouch:(UITouch *)touch forEvent:(UIEvent *)event recognize
     return touch == _currentTouch && touch._isPointerTouch;
 }
 
-- (WebKit::NativeWebMouseEvent *)lastMouseEvent
+- (WebKit::NativeWebMouseEvent *)prepareToDispatchMouseEvent
```

Comment:
something about this name feels weird to me.  i wouldn't expect it to have a return value based on the name (i.e. it feels more like a "hey object please do this thing" rather than a "hey object give me a value").

also, this new logic also means that it can't be called twice in a row (such as if someone doesn't wanna save the return value to a variable the first time (e.g. how `WebPageProxy::currentlyProcessedMouseDownEvent` is used).  i feel like we should move this logic to some other method (e.g. `shouldHandleEvent`) so that we don't prevent `_lastEvent` from being accessed